tests/manager/manager-executeCommand-004.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+MongoDB\Driver\Manager::executeCommand() options
```

Comment:
This file was renamed from `manager-executeRawCommand-001.phpt`

Reply:
I had specifically named this to match that PR's changes - where we have Raw, Read, Write, and ReadWrite, so I am not sure if this a good idea.

Reply:
Was this the only occurrence of "Raw" from that PR? I gathered it was more confusing to have a test named in the `class-method` for a method that doesn't exist.

We both know that `executeCommand()` implies `MONGO_CMD_RAW`. Would adding "Raw Command" (or similar) to the test title be a fair compromise?

Reply:
Just noticed that `manager-executeCommand_error-002.phpt` was added in the same PR and didn't use the "RawCommand" name.